.gitignore

```diff
@@ -8,3 +8,5 @@ npm-debug.log
 # VIM
```

Comment:
Can you please clarify in your commit message what happens to `moment.js` in this commit?
Please also remove the duplicate `bundle css` line.

Reply:
Sorry I not clear what you mean. Which file/lines are you revering to?

Reply:
I was just referring to the details text in your **commit message**.

Reply:
@mojoaxel What I meant is to rewrite/clean up your commit message.